deploy/eck-operator/values.yaml

```diff
@@ -199,13 +199,13 @@ podMonitor:
   podMetricsEndpointConfig: {}
   # honorTimestamps: true
 
-# Internal use only
-internal:
+# Globals meant for internal use only
+global:
```

Comment:
Should we also update the following test files in `hack/manifest-gen/testdata/` ?
* [`no_defaults.yaml`](https://github.com/elastic/cloud-on-k8s/blob/5362e7317d240b6313d7cec32d2dd8f3743f36bf/hack/manifest-gen/testdata/no_defaults.yaml#L179)
* [`kube113.yaml`](https://github.com/elastic/cloud-on-k8s/blob/5362e7317d240b6313d7cec32d2dd8f3743f36bf/hack/manifest-gen/testdata/kube113.yaml#L160)
* [`kube116.yaml`](https://github.com/elastic/cloud-on-k8s/blob/5362e7317d240b6313d7cec32d2dd8f3743f36bf/hack/manifest-gen/testdata/kube116.yaml#L160)

If yes I think I would expect the tests to fail, but I think the purpose is only to validate the schemas using `garethr/kubeval`.

Reply:
Good catch. Indeed we don't have a good way of verifying that the test input is what we expect. I don't have a good idea right now.